src/views/MondoMarket/index.jsx

```
@@ -0,0 +1,228 @@
+      {/* All Regions */}
+      <div className="flex items-center gap-5 overflow-x-auto flex-1 scrollbar-hidden">
+        {regions?.map((region) => (
+          <div
+            key={region.id}
+            className={` ${
+              activeRegion?.name === region?.name
+                ? `${bgColor} text-white`
+                : "border border-gray-300 text-dark_bg_5 dark:text-dark_text_1"
+            } px-2 py-1 rounded-md  cursor-pointer min-w-fit`}
+            onClick={() => setActiveRegion(region)}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)